packages/netlify-cms-backend-bitbucket/src/implementation.ts

```diff
@@ -462,31 +462,26 @@ export default class BitbucketBackend implements Implementation {
     });
   }
 
-  loadMediaFile(branch: string, file: UnpublishedEntryMediaFile) {
-    const readFile = (
+  async loadMediaFile(path: string, id: string, { branch }: { branch: string }) {
```

Comment:
aligned the signature with other methods and use `async/await` (easier to debug)